math/perfectnumber_test.go

```diff
@@ -0,0 +1,144 @@
+package math
+
+import (
+	"testing"
+)
+
+//example data verified with https://oeis.org/A001065/b001065.txt
```

Comment:
The comments styling in Go is very specific. Please read up on it [here](https://tip.golang.org/doc/comment)